src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -233,12 +241,12 @@ private bool AllowClientConnection(TcpClient tcpClient)
             if (endpointCanBeWhiteListed)
             {
                 this.logger.LogTrace("(-)[ENDPOINT_WHITELISTED_ALLOW_CONNECTION]:true");
-                return true;
+                return (true, "Endpoint Whitelisted, Allow Connection even if in IBD.");
```

Comment:
Change the string to: `Inbound Accepted: Endpoint Whitelisted.`

Reply:
the concept here is that the endpoint connected even if it was in IBD, because the other peer is whitelisted, the message isn't correct, I changed it to
Inbound Accepted: Whitelisted endpoint connected during IBD.

Reply:
nvm I kept your suggestion, it's fine